Week3/homework/6-step3.js

```diff
@@ -3,4 +3,32 @@
 const arr2d = [[1, 2], [3, 4], [5, 6]];
 const arr3d = [[[1, 2], [3, 4]], [[5, 6], [7, 8]]];
 
-// add your solution here, or add a comment on how you would tackle this problem
+function printArray(arr) {
+
+  // if arr is an array call printArray for each element else print it.
+  if (Array.isArray(arr)) {
+    for (let i = 0; i < arr.length; i++) {
+      printArray(arr[i]);
+    }
+  } else console.log(arr);
```

Comment:
Nice work.  
One style point here.  While what you wrote is legal and interpreted correctly by the interpretor, it is confusing to read.  I would suggest wrapping the else statement in {} as well.